packit/config/common_package_config.py

```diff
@@ -165,6 +165,8 @@ class CommonPackageConfig:
             the corresponding macro.
         status_name_template: Template for configurable names for status checks.
         sig: Special interest group (SIG) that maintains the “downstream” package.
+        differential_scan_after_copr_build: Whether to run a differential scan after
```

Comment:
Might be a better name would be `osh_diff_scan_after_copr_build`, just in case this config option should not be confused with any other scanning utility in the future.

Reply:
I believe we have discussed there are no plans for integrating with other scanners, but I will bring this up at our meeting on Monday and change if needed, this is now not used anywhere yet :) thanks for the feedback!

Reply:
We only run open source scanners in public instance of OpenScanHub, and I would not be surprised if people request for `scan-in-coverity` option if they see Coverity missing in OSH. You should consider that for the future. Thanks!

Reply:
We have decided we will go with your suggestion, even if it is likely the config will be changed in the future for more complex functionalities (opened #2359). Thanks again!